src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2664,7 +2664,7 @@ public static void IsIanaIdTest()
         {
             bool expected = !s_isWindows;
 
-            Assert.Equal((expected || TimeZoneInfo.Local.Id.Equals("Utc", StringComparison.OrdinalIgnoreCase)), TimeZoneInfo.Local.HasIanaId);
+            Assert.Equal((expected || PlatformDetection.IsiOS || PlatformDetection.IstvOS || TimeZoneInfo.Local.Id.Equals("Utc", StringComparison.OrdinalIgnoreCase)), TimeZoneInfo.Local.HasIanaId);
```

Comment:
That doesn't make sense to me. I'd assume that `!s_isWindows` is `true` thus `expected == true` and this condition would not need the change.

Reply:
Ah oops, I got the order wrong when I checked the failure